commit 2bf45d3c6e585eac73edec442ef54c55c341884b (refs/changes/01/5701/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2019-03-04T20:14:11+01:00 (7 months ago)
    
    TRITON-1283 sdcadm `make check` broken
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 3c1196d..4d68768 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,11 @@
 
 # sdcadm Changelog
 
+## 1.24.7
+
+- TRITON-1276 sdcadm experimental update-gz-tools --latest --just-download dumps core
+- TRITON-1272 sdcadm test failures due to outdated tape version
+
 ## 1.24.6
 
 - TRITON-1236 steps.servers.selectServers for convenient -s/-S opt handling,
